src/entities/ValueNode/ValueNode.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { createValueNodeMock } from '../../utils/mocks/createValueNodeMock';
+import { BlockNode, createBlockNodeName } from '../BlockNode';
+import { createBlockNodeMock } from '../../utils/mocks/createBlockNodeMock';
+import { EditorDocument } from '../EditorDocument';
+
+describe('ValueNode', () => {
+  describe('.update()', () => {
+    it('should add new data field associated with this value node', () => {
```

Comment:
Are you sure that we need to allow adding new properties to ValueNode? 

Seems like ValueNode should only allow updating of existed value which was set through the Constructor. And disallow adding new properties later